backend/api/Areas/Leases/Mapping/Lease/RegionMap.cs

```diff
@@ -0,0 +1,16 @@
+using Mapster;
```

Comment:
Hmm, does the existing type code mapping not work for regions? It looks like it should, in which case you can add/update the region partial class to extend : ITypeEntity<string>

Reply:
Sadly not since PimsRegion attribute names are not standard like a typecode